lib/board.rb

```diff
@@ -0,0 +1,55 @@
+class Board
+  DIRECTIONS = [:NORTH, :EAST, :SOUTH, :WEST]
+
+  attr_accessor :x_pos
+  attr_accessor :y_pos
+  attr_accessor :f_index
+
+  def initialize(x_size, y_size)
+    @x_size = x_size
+    @y_size = y_size
+  end
+
+  def place(x_pos, y_pos, f)
+    x_pos = validate_pos(x_pos, @x_size)
+    y_pos = validate_pos(y_pos, @y_size)
+    f_index = validate_f(f)
+
+    @x_pos = x_pos
+    @y_pos = y_pos
+    @f_index = f_index
+
+    @initialized = true
+  end
+
+  def initialized?
```

Comment:
Yeah, considered calling it 'placed' but it sounded weird... Oh well, if you think it's better :)